Disabled housekeeping timer until we have libffi on Android.

Otherwise causes crash due to NSInvocation requiring libffi/libcall.

diff --git a/Source/NSRunLoop.m b/Source/NSRunLoop.m
index 1746fa3d0..c85703bac 100644
--- a/Source/NSRunLoop.m
+++ b/Source/NSRunLoop.m
@@ -744,6 +744,7 @@ + (NSRunLoop*) _runLoopForThread: (NSThread*) aThread
   if (nil == current)
     {
       current = info->loop = [[self alloc] _init];
+#if 0 // disabled housekeeping timer until we have libffi on Android
       /* If this is the main thread, set up a housekeeping timer.
        */
       if (nil != current && [GSCurrentThread() isMainThread] == YES)
@@ -788,6 +789,7 @@ + (NSRunLoop*) _runLoopForThread: (NSThread*) aThread
           #endif
           [arp drain];
         }
+#endif
     }
   return current;
 }
